openpype/hosts/maya/plugins/publish/extract_camera_mayaScene.py

```diff
@@ -131,15 +134,14 @@ def process(self, instance):
                              "bake to world space is ignored...")
 
         # get cameras
-        members = cmds.ls(instance.data['setMembers'], leaf=True, shapes=True,
-                          long=True, dag=True)
-        cameras = cmds.ls(members, leaf=True, shapes=True, long=True,
-                          dag=True, type="camera")
+        members = set(cmds.ls(instance.data['setMembers'], leaf=True,
+                      shapes=True, long=True, dag=True))
+        cameras = set(cmds.ls(members, leaf=True, shapes=True, long=True,
+                      dag=True, type="camera"))
 
         # validate required settings
         assert isinstance(step, float), "Step must be a float value"
-        camera = cameras[0]
-        transform = cmds.listRelatives(camera, parent=True, fullPath=True)
+        transform = cmds.listRelatives(cameras, parent=True, fullPath=True)
```

Comment:
Note that older versions of Maya can't deal receiving `set` as inputs. I believe it's Maya 2020 and below, but not entirely sure.